### DISCUSSION TITLE
技术小白可以付费搭建吗？

### DISCUSSION THREAD

**Discussion by zjb500** [AlistGo/alist#313] [Q&A] [answered]
不懂程序方面的知识，流程看下来很多不懂，比如这条：赋予程序执行权限：chmod +x alist-xxxx
无论是字面意思还是操作方法都没理解。
所以可以付费请哪位大神搭建吗？谢谢。

**Comment by xhofe** [comment, marked as answer]
通过邮件与我联系：support@nn.ci
